backend-lib/src/main/rust/lib.rs

```diff
@@ -1050,7 +1050,7 @@ pub extern "C" fn Java_cash_z_ecc_android_sdk_internal_jni_RustBackend_writeBloc
     _: JClass<'local>,
     db_cache: JString<'local>,
     block_meta: JObjectArray<'local>,
-) -> jboolean {
```

Comment:
The same argument applies to `initBlockMetaDb`: the return value is always discarded.